provisioning_templates/PXELinux/coreos_pxelinux.erb

```diff
@@ -6,7 +6,13 @@ oses:
 - CoreOS
 %>
 DEFAULT coreos
+PROMPT 1
+TIMEOUT 15
+
+DISPLAY boot.msg
```

Comment:
This `boot.msg` file, is it part of a different patch in our installer? Because it won't be present by default. I think you may not want to include this command here.